front/public/conf/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
-  "short_name": "Elm App",
-  "name": "Create Elm App Sample",
+  "short_name": "OpenBoard",
```

Comment:
Is the camel case a sure thing?

Reply:
@champagneabuelo Is it "Open Board", "Openboard, "OpenBoard", "openboard"... ? It's "switchboard"

Reply:
`"name": "Create Elm App Sample",` needs to be updated anyway.

Reply:
I marked this as resolved as it pertains to this PR.

Reply:
@nqthqn ive been using "openboard", but i wouldnt mind "Openboard"